css-layout-classes/styles.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
you don't need display flex or flex wrap for the container here. the rows and columns will handle that for this one!